File: lab02/Reporte.java
public class Reporte {
    
    public void generarReporteEstudiante(Estudiante estudiante){
        System.out.println("Reporte del Estudiante: ");
        System.out.println("Nombre: "+estudiante.getNombre());
        System.out.println("Edad: "+estudiante.getEdad()+" años");
        System.out.println("Carrera: "+estudiante.getCarrera());
    }
}

File: lab02/Reporte.py
from Estudiante import Estudiante

class Reporte:

    def generar_reporte_estudiante(self, estudiante: Estudiante):
        print(f"Nombre: {estudiante.nombre}")
        print(f"Edad: {estudiante.edad} años")
        print(f"Carrera: {estudiante.carrera}")

    def __str__(self) -> str:
        return "Módulo de generación de reportes académicos."